scripts/load_kube_credentials.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Function to check if a command exists
+not_installed() {
+    ! type "$1" &> /dev/null
+}
+
+# Check for required commands and provide installation links
+for cmd in jq yq op; do
+    if not_installed "$cmd"; then
+        echo "$cmd is not installed. You can install it from: ${INSTALLATION_LINKS[$cmd]}"
+        exit 1
+    fi
+done
+
+# The string to match with context.user
```

Comment:
NITPICK: We could update the docstring with only context

Reply:
Yup 🙂!
Updated by: [ebd5b75](https://github.com/OpenMined/PySyft/pull/8199/commits/ebd5b7516ff3d1472a0fd035f074b3f1a7b414b8)